yautpa-desktop/src/components/Countdown/index.scss

```diff
@@ -1,20 +1,26 @@
 @mixin button() {
-    color: white;
-    background-color: black;
-    border: white 1px solid;
-    border-radius: 16%;
+    color: #0051ff;
+    background-color: #ffffff;
+    border: #0051ff .25rem solid;
     margin-bottom: 0.5rem;
-    padding: 0.2rem 0.4rem 0.2rem 0.4rem
+    margin-left: 10rem;
```

Comment:
This margin-left in the mixin gets overwritten in both use cases in line 16 and line 23